main.py

```diff
@@ -0,0 +1,53 @@
+from copy import deepcopy
+
+def primary_forbidden_sequences(list_of_all_forbidden_sequences,P):
+    all_sequences = deepcopy(list_of_all_forbidden_sequences)
+    index_to_be_taken = [0]*P
+    for i in range(P):
+        if index_to_be_taken[i] == 0:
+            current_sequence = all_sequences[i]
+            n = len(current_sequence)
+            working_sequences = deepcopy(all_sequences)
+            for j in range(n):
```

Comment:
As `n` is only used here I think, it might be simpler just to write `for j in range(len(current_sequence)):`